### PR TITLE
Add KIP-14 & KIP-15 details

### PR DESCRIPTION
**Pull request by ssreelekhya**
Still need to fill in start and end timestamp and KNC transfer address

### CODE REVIEW

**Review by ShaneKyber — APPROVED** [review]

**Review by ShaneKyber — APPROVED** [review]

File: KIPs/kip-15.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+Based on forum discussions, the Kyber community is proposing that the following projects be included in the `Rainmaker` Joint Liquidity Mining program:
+
+1. Creator chain on Ethereum, for [CTR-ETH](https://gov.kyber.org/t/creator-ctr-joint-liquidity-mining-proposal-23/309/29)
+
+2. xToken on Ethereum, for [XTK-ETH](https://gov.kyber.org/t/joint-liquidity-mining-w-xtoken-on-kyberdmm/355)
+
+3. xDollar on Polygon in 2 phases, for [xUSD-USDC](https://gov.kyber.org/t/joint-liquidity-mining-with-xdollar-polygon-on-kyberdmm/379)
+
+4. 4. Step Hero on BSC, for the [HERO token](https://gov.kyber.org/t/stephero-hero-joint-liquidity-mining-on-kyberdmm/397 )
```
Comment (manhlx3006) [inline KIPs/kip-15.md:30]: `4. 4. Step Hero on BSC`

**Review by Anyhowclick — APPROVED** [review]

File: KIPs/kip-15.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+status: Active
+type: Spend KNC
+authors: Kyber Network team
+created: 2021-09-06
+---
+
+```
+executor: short timelock executor
+voting_power: current staked voting power
+execution: transfer 95,350 knc to 0xD69D7A1031e6E63a162414F9A77278757690C30E (kyber multisig to handle KNC bridging between ethereum and other chains)
```
Comment (manhlx3006) [inline KIPs/kip-15.md:13]: The execution should depend on how many projects are accepted?

**Review by loiluu — APPROVED** [review]

File: KIPs/kip-15.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+start_timestamp: 1630998000 (Tue, 7 Sep 2021 07:00:00 UTC)
+end_timestamp: 1631603227 (Tue, 14 Sep 2021 07:07:07 UTC)
```
Comment (manhlx3006) [inline KIPs/kip-15.md:15]: this is the start time of new epoch, please reduce it by few seconds.

**Review by manhlx3006 — CHANGES_REQUESTED** [review]

File: KIPs/kip-15.md (hunk 3 above)
Comment (ssreelekhya) [reply KIPs/kip-15.md:15]: Sure

Comment (manhlx3006) [reply KIPs/kip-15.md:15]: Can use this contract `0xfbd73581af42b26295CF4B88B74A5AF71F209F8b` and call function `validateProposalCreation(startTime, endTime)` to validate if start & end times are valid to create a new proposal

File: KIPs/kip-14.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+status: Active
+type: Mint KNC
+authors: Kyber Network team
+created: 2021-09-06
+---
+
+```
+voting_campaign_type: general
+voting_start_timestamp: 1631026800 (Tue, 7 Sep 2021 07:00:00 UTC)
+voting_end_timestamp: 1631603227 (Tue, 14 Sep 2021 07:07:07 UTC)
```
Comment (manhlx3006) [inline KIPs/kip-14.md:13]: Similar here, end time should be a lower value

File: KIPs/kip-15.md (hunk 2 above)
Comment (ssreelekhya) [reply KIPs/kip-15.md:13]: Is it okay if we say "transfer knc to 0xD69D7A1031e6E63a162414F9A77278757690C30E, depending on projects accepted"

File: KIPs/kip-14.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+kip: 14
+title: Allocate additional 5M KNC from the ecosystem fund
+status: Active
+type: Mint KNC
+authors: Kyber Network team
+created: 2021-09-06
+---
+
+```
+voting_campaign_type: general
```
Comment (manhlx3006) [inline KIPs/kip-14.md:11]: should be `binary`?

File: KIPs/kip-14.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+type: Mint KNC
+authors: Kyber Network team
+created: 2021-09-06
+---
+
+```
+campaign_type: binary
+executor: short timelock executor
+voting_power: current staked voting power
+execution: transfer 5M KNC from ecosystem funds to daoOperator
```
[the remark this answers is not in the document]
Comment (manhlx3006) [reply KIPs/kip-14.md:14]: Not sure if we need to change the title of the KIP-14

[the remark this answers is not in the document]
Comment (ssreelekhya) [reply KIPs/kip-14.md:14]: Oh! Can we say like this
transfer 5M KNC from ecosystem funds to daoOperator & 2M to kyberDao multisig

[the remark this answers is not in the document]
Comment (ssreelekhya) [reply KIPs/kip-14.md:14]: or 
transfer 5M KNC from ecosystem funds to daoOperator & 2M for Avalaunch LM

**Review by ShaneKyber — APPROVED** [review]

**Review by manhlx3006 — APPROVED** [review]

**Pull request merged**